tests/storage/linstor/conftest.py

```diff
@@ -43,17 +43,30 @@ def lvm_disks(host, sr_disks_for_all_hosts, provisioning_type):
 
 @pytest.fixture(scope="package")
 def storage_pool_name(provisioning_type):
+    # FIXME: this needs an explanation
     return GROUP_NAME if provisioning_type == "thick" else STORAGE_POOL_NAME
 
+# FIXME why having this feature of session scope?  Shouldn't it make
+# it impossible to run both thin and thick tests in the same session?
```

Comment:
Since all linstor tests were going to be repeated with just one change of SR type, pytest `params` technique was used.